src/main/routes/confirmDeleteItem.ts

```diff
@@ -8,6 +8,7 @@ router.get("/deleteitem", (req, res, next) => {
   if (req.adminWebAuthorization && req.adminWebAuthorization.canManageDefinition) {
     const responseContent: { [k: string]: any } = {};
     responseContent.adminWebAuthorization = req.adminWebAuthorization;
+    responseContent.user = JSON.stringify(req.authentication.user);
```

Comment:
can this be unit tested?

Reply:
@gbenadikar - I've amended the tests to check for the presence of the user data within a hidden `input` on the HTML page